docs/quick-visual-tutorials/visual-k8s-cluster-mapping.mdx

```diff
@@ -143,17 +143,15 @@ We also (as a default) told Otterize Cloud that there is a global default-deny n
 
 #### Blocking status
 
-Note that the locks themselves are green, indicating that you could now turn on enforcement without blocking any clients.
+Note that the locks themselves are yellow, indicating that you could now turn on enforcement and blocking not intented clients.
 
-Similarly, all the thick connecting lines between the services are green: none of these client calls would be blocked if enforcement were turned on. If one were red, that would tell you it would be blocked, as you might have guessed.
+Similarly, all the thick connecting lines between the services are yellow: Client calls would be blocked if enforcement were turned on. If one were red, that would tell you it is blocked, as you might have guessed.
 
-But why would these clients not be blocked if enforcement were on &mdash; doesn't that mean the services they call would not be protected? Yes, and the access graph lets you know that too.
+Click on a service, e.g. the payment service:
```

Comment:
Add the edge I requested in the above comment, and replace this service. As it is the product (not the guide) is confusing, since it says enforcement would block, but it actually wouldn't since there are no intents. Enforcement would still be off for this first since only the first intent would cause it to get enabled.

So we need to either explain that here (or in the cloud, but this is out of scope for this PR), or not show it (which I think is better), and instead show the edge which has a better explanation.